lumigator/python/mzai/schemas/lumigator_schemas/experiments.py

```diff
@@ -22,7 +20,6 @@ class ExperimentResponse(BaseModel, from_attributes=True):
     id: UUID
     name: str
     description: str
-    status: JobStatus
```

Comment:
this isn't being used at the experiment level

Reply:
It'll be introduced shortly, btw.

Reply:
@javiermtorres / @veekaybee does this mean [this doc comment](https://github.com/mozilla-ai/lumigator/blob/1c73e22cc815bf492017a24e1dedca8216f2cb88/lumigator/python/mzai/backend/backend/services/experiments.py#L123) doesn't need adjusting if `status` is coming back?